contracts/Swap/interfaces/IPancakeSwapV2Router.sol

```diff
@@ -1,75 +1,6 @@
 pragma solidity 0.8.13;
 
 interface IPancakeSwapV2Router {
-    function swapExactTokensForTokens(
```

Comment:
We need to keep these "redundant" functions because our frontend will interact only with this SwapRouter contract, not directly with PCS